app/components/task_list_items/assessment/consultees_consulted_component.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  module Assessment
+    class ConsulteesConsultedComponent < TaskListItems::BaseComponent
+      def initialize(planning_application:)
+        @planning_application = planning_application
+      end
+
+      private
+
+      attr_reader :planning_application
+
+      def link_text
+        "Check consultees consulted"
```

Comment:
Can you put this in the .yml file, that's what we've been doing (I also always get told to do it)